lib/Service/ChecksumVerificationService.php

```diff
@@ -24,24 +26,24 @@
 
 use OCA\Talk\Exceptions\UnauthorizedException;
 
-class SIPBridgeService {
+class ChecksumVerificationService {
 	/**
 	 * Check if the current request is coming from an allowed backend.
 	 *
-	 * The SIP bridge is sending the custom header "Talk-SIPBridge-Random"
+	 * The backend servers are sending custom headers "Talk-…-Random"
 	 * containing at least 32 bytes random data, and the header
-	 * "Talk-SIPBridge-Checksum", which is the SHA256-HMAC of the random data
+	 * "Talk-…-Checksum", which is the SHA256-HMAC of the random data
```

Comment:
Only to clarify more the meaning of ... using the pattern that all inside a tag is mandatory

```suggestion
	 * "Talk-<featureName>-Checksum", which is the SHA256-HMAC of the random data
```